galaxychop/galaxychop.py

```diff
@@ -148,13 +143,21 @@ class Galaxy:
     pot_dark = np.loadtxt(path_potencial+'dark_ID_'+str(ID[j])+'.dat')
     pot_gas = np.loadtxt(path_potencial+'gas_ID_'+str(ID[j])+'.dat')
 
-    # Calculamos la energia.
-    E_tot_star = k_star - pot_star[:, 1]
-    E_tot_dark = k_dark - pot_dark[:, 1]
-    E_tot_gas = k_gas - pot_gas[:, 1]
+    def energy(self, pot_star, pot_dark, pot_gas):
+
+        '''Calculamos la energia cinetica y potencial y total'''
+
+        k_s = 0.5*(self.vx_s**2 + self.vy_s**2 + self.vz_s**2)
+        k_dm = 0.5*(self.vx_dm**2 + self.vy_dm**2 + self.vz_dm**2)
+        k_g = 0.5*(self.vx_g**2 + self.vy_g**2 + self.vz_g**2)
```

Comment:
Check style.
Basically use spaces around operators.
Probably putting the gas kinetic calculation in the middle is more appealing to the eye.

Reply:
I agree with this. In this way, the signs and operators are aligned for stars and gas.

Reply:
jaja. I changed the order.